pinterest/teletraan
# Modifications to render error message bar for Scheduled Actions and Scaling Alarms

emit useful logs and display a failure page when scheduled action is below min cluster size or greater than max cluster size and when invalid Scaling Alarm is saved.

Test Plan:

1. Navigate to Autoscaling Settings page
2. Create a a Scheduled Action with a valid scheduled (i.e 0 1 * * *) and a Capacity > the Max Size of the Auto Scaling Group or < the Min Size of the Auto Scaling Group
<img width="866" alt="Screen Shot 2022-08-26 at 4 49 05 PM" src="https://user-images.githubusercontent.com/69278532/187004936-808df0b5-3897-4544-ba19-04d9d43883d2.png">
3. Confirm that you are taken to an error page upon saving
<img width="865" alt="Screen Shot 2022-08-26 at 4 49 18 PM" src="https://user-images.githubusercontent.com/69278532/187004955-58c5ba99-02d9-4799-87f6-61b0de4736c3.png">
4. Return to the Autoscaling Settings Page
5. Creating a Scaling Alarm with a negative values under "When the metric:"
<img width="865" alt="Screen Shot 2022-08-26 at 4 50 42 PM" src="https://user-images.githubusercontent.com/69278532/187004982-61b5708d-1c87-40b3-875c-aac52af2d114.png">
6. Confirm that you are taken to an error page upon saving
<img width="869" alt="Screen Shot 2022-08-26 at 4 50 58 PM" src="https://user-images.githubusercontent.com/69278532/187005038-77741e6b-f9cc-4def-97e7-59313e739f3a.png">
